frontend/src/Frontend.hs

```diff
@@ -139,6 +139,7 @@ newHead routeText = do
   ss "https://fonts.googleapis.com/css?family=Work+Sans"
   ss (static @"css/font-awesome.min.css")
   ss (static @"css/ace-theme-chainweaver.css")
+  js "/static/externs.js"
```

Comment:
Is it really necessary to import this file in the head? I thought it is is only needed as an input to the closureCompiler step

Reply:
Removed.